vite.config.ts

```diff
@@ -55,4 +55,7 @@ export default defineConfig({
       util: 'util',
     },
   },
+  server: {
+    port: 3000,
+  },
```

Comment:
By the way, please add `--open` to open the browser when starting (Ref: https://vitejs.dev/guide/cli.html#options)